gammapy/data/filters.py

```diff
@@ -97,3 +98,14 @@ def _filter_by_time(self, data):
     def copy(self):
         """Copy the `ObservationFilter` object."""
         return copy.deepcopy(self)
+
+    @staticmethod
```

Comment:
I think it would safer to make this a `property` and not define the value on init.